src/auth/logout.ts

```diff
@@ -3,6 +3,9 @@ import store from '../redux/store';
 import { apiError } from './redux';
 
 export default function(): void {
+  // Reset birthDate cookie here. This way we will never run to a problem
+  // where user is redirected to registration from without or with wrong birthDate
+  document.cookie = 'birthDate=';
```

Comment:
Cookies are not my forte, but what happens to other cookies when you set this?

Reply:
They are still there, document.cookie looks like a normal string but it behaves like name-value pair. When setting `document.cookie  =` you would think it replaces the whole cookie string, but instead just adds the value to existing string.